react/src/components/atoms/forms/SelectBox/index.js

```diff
@@ -34,17 +35,27 @@ class SelectBox extends React.Component {
   }
 
   render() {
-    let classNames = '';
-    if (this.props.className) {
-      classNames = this.props.className;
-    } else {
-      classNames = !this.props.required ? 'ma__select-box js-dropdown ma__select-box--optional' : 'ma__select-box js-dropdown';
-    }
-    const selectClassNames = this.props.required ? 'ma__select-box__select js-dropdown-select js-required' : 'ma__select-box__select js-dropdown-select';
     const { selected } = this.state;
     const { stackLabel } = this.props;
-    const labelClassNames = stackLabel ? 'ma__select-box__label' : 'ma__label--inline ma__label--small';
-    const selectBoxInline = stackLabel ? '' : 'ma__select-box__field--inline';
+    const sectionClasses = classNames({
```

Comment:
The old logic seems to be that if this.props.className exists, then only use that for the section classes.

This should probably be wrapped in an if/else so that if this.props.className exists, sectionClasses is assigned to be this.props.className.